engine/graphics/src/graphics.h

```diff
@@ -341,6 +343,27 @@ namespace dmGraphics
         uint32_t            m_StructSize;
     };
 
+    struct VertexDeclaration
```

Comment:
We have merged the representation for vertex declarations into a shared struct. If it works for opengl and vulkan, it will work for all adapter types